### PR TITLE
startup: Change Location to Project in first-time user messages

### PR DESCRIPTION
**Pull request by ecodiv**
The startup message for first-time users refers to the old Locations. This should be changed to Projects to match the new naming used in 8.4+.

### CODE REVIEW

**Comment by echoix** [comment]
@petrasovaa Did we settle on lowercase or uppercase project finally?

**Comment by echoix** [comment]
There already has been a long discuss on this, we don't need to open it back. I just don't remember the conclusion

**Comment by veroandreo** [comment]
> @petrasovaa Did we settle on lowercase or uppercase project finally?

Thanks Paulo @ecodiv ! We missed that one

We are using lowercase for "project" unless the word is the start of a sentence of course. And since you are at it please also use lowercase for mapset. Thanks! :raised_hands:

**Comment by ecodiv** [comment]
> There already has been a long discuss on this, we don't need to open it back. I just don't remember the conclusion

OK, good to know for how to use it in tutorials.

**Comment by ecodiv** [comment]
>  And since you are at it please also use lowercase for mapset. Thanks! 🙌

Done

File: gui/wxpython/datacatalog/infomanager.py [hunk 1]
```diff
@@ -33,16 +33,16 @@ def __init__(self, infobar, giface):
     def ShowDataStructureInfo(self, onCreateLocationHandler):
         """Show info about the data hierarchy focused on the first-time user"""
         buttons = [
-            (_("Create new Location"), onCreateLocationHandler),
+            (_("Create new Project"), onCreateLocationHandler),
```
Comment (echoix) [inline gui/wxpython/datacatalog/infomanager.py:36]:
```suggestion
            (_("Create new project"), onCreateLocationHandler),
```

File: gui/wxpython/datacatalog/infomanager.py [hunk 2]
```diff
@@ -33,16 +33,16 @@ def __init__(self, infobar, giface):
...
             (_("Learn more"), self._onLearnMore),
         ]
         message = _(
-            "GRASS GIS helps you organize your data using Locations (projects) "
-            "which contain Mapsets (subprojects). All data in one Location is "
+            "GRASS GIS helps you organize your data using Projects (locations) "
+            "which contain mapsets (subprojects). All data in one Project is "
```
Comment (echoix) [inline gui/wxpython/datacatalog/infomanager.py:41]:
```suggestion
            "GRASS GIS helps you organize your data using projects (locations) "
            "which contain mapsets (subprojects). All data in one project is "
```

File: gui/wxpython/datacatalog/infomanager.py [hunk 3]
```diff
@@ -33,16 +33,16 @@ def __init__(self, infobar, giface):
...
             "in the same coordinate reference system (CRS).\n\n"
-            "You are currently in Mapset PERMANENT in default Location {loc} "
+            "You are currently in mapset PERMANENT in default Project {loc} "
```
Comment (echoix) [inline gui/wxpython/datacatalog/infomanager.py:43]:
```suggestion
            "You are currently in mapset PERMANENT in default project {loc} "
```

File: gui/wxpython/datacatalog/infomanager.py [hunk 4]
```diff
@@ -33,16 +33,16 @@ def __init__(self, infobar, giface):
...
             "which uses WGS 84 (EPSG:4326). "
-            "Consider creating a new Location with a CRS "
+            "Consider creating a new Project with a CRS "
```
Comment (echoix) [inline gui/wxpython/datacatalog/infomanager.py:45]:
```suggestion
            "Consider creating a new project with a CRS "
```

File: gui/wxpython/datacatalog/infomanager.py [hunk 5]
```diff
@@ -55,11 +55,11 @@ def ShowImportDataInfo(self, OnImportOgrLayersHandler, OnImportGdalLayersHandler
             (_("Import raster data"), OnImportGdalLayersHandler),
         ]
         message = _(
-            "You have successfully created a new Location {loc}. "
-            "Currently you are in its PERMANENT Mapset which is used for "
+            "You have successfully created a new Project {loc}. "
+            "Currently you are in its PERMANENT mapset which is used for "
```
Comment (echoix) [inline gui/wxpython/datacatalog/infomanager.py:59]:
```suggestion
            "You have successfully created a new project {loc}. "
            "Currently you are in its PERMANENT mapset which is used for "
```

File: gui/wxpython/datacatalog/infomanager.py [hunk 6]
```diff
@@ -55,11 +55,11 @@ def ShowImportDataInfo(self, OnImportOgrLayersHandler, OnImportGdalLayersHandler
...
             "storing your base maps to make them readily available in other "
-            "Mapsets. You can create new Mapsets for different tasks by right "
-            "clicking on the Location name.\n\n"
+            "mapsets. You can create new mapsets for different tasks by right "
+            "clicking on the Project name.\n\n"
```
Comment (echoix) [inline gui/wxpython/datacatalog/infomanager.py:62]:
```suggestion
            "clicking on the project name.\n\n"
```

File: gui/wxpython/datacatalog/infomanager.py [hunk 7]
```diff
@@ -81,8 +81,8 @@ def ShowFallbackSessionInfo(self, reason_id):
         """Show info when last used mapset is not usable"""
         string = self._text_from_reason_id(reason_id)
         message = _(
-            "{string} GRASS GIS has started in a temporary Location. "
-            "To continue, use Data Catalog below to switch to a different Location."
+            "{string} GRASS GIS has started in a temporary Project. "
+            "To continue, use Data Catalog below to switch to a different Project."
```
Comment (echoix) [inline gui/wxpython/datacatalog/infomanager.py:85]:
```suggestion
            "{string} GRASS GIS has started in a temporary project. "
            "To continue, use Data Catalog below to switch to a different project."
```

File: gui/wxpython/datacatalog/infomanager.py [hunk 8]
```diff
@@ -94,8 +94,8 @@ def ShowLockedMapsetInfo(self, OnSwitchMapsetHandler):
         buttons = [(_("Switch to last used mapset"), OnSwitchMapsetHandler)]
         message = _(
             "Last used mapset in path '{mapsetpath}' is currently in use. "
-            "GRASS GIS has started in a temporary Location. "
-            "To continue, use Data Catalog below to switch to a different Location "
+            "GRASS GIS has started in a temporary Project. "
+            "To continue, use Data Catalog below to switch to a different Project "
```
Comment (echoix) [inline gui/wxpython/datacatalog/infomanager.py:98]:
```suggestion
            "GRASS GIS has started in a temporary project. "
            "To continue, use Data Catalog below to switch to a different project "
```

**Review by echoix — CHANGES_REQUESTED** [review]
If we want lowercased project, the following would need to be adjusted

**Comment by ecodiv** [comment]
> If we want lowercased project, the following would need to be adjusted

Committed the suggested changes

**Review by petrasovaa — APPROVED** [review]

**Review by echoix — APPROVED** [review]

**Pull request merged**